main.py

```diff
@@ -1,77 +1,112 @@
-''' Simple aiohttp login system with default user!:
-This system is built with aiohttp module.
-Required Python-3 or 3+
-'''
+#!/usr/bin/env python
+# 
+# Simple aiohttp login system with default user!.
+# This system is built with aiohttp module.
+# Requires Python3 or higher.  
 
+
+import os
+import os.path
 from aiohttp import web
 
-# Customize the cookie name, value and cookie expire date
+
+# Customize the cookie name, value and cookie expire date. 
+# The cookie will be stored 30 days long.
 COOKIE_NAME = "OldTamil"
 COOKIE_VALUE = 0
-COOKIE_LIFE_TIME = 24*60*60*30
+COOKIE_LIFE_TIME = 24 * 60 * 60 * 30   
+
 
-# User credentials and here we could use more users in future.
-User ='Hippod'
-Password = 'admin'
+# User credentials.
+USERNAME = 'Hippod'
+PASSWORD = 'admin'
 
-# Define the required host and port
+# Define the required host and port.
 HOST = '127.0.0.1'
-PORT = 8080
+PORT = 8888
+
+
+def check_cookie(request):
+    """cookie_check!
+    This function will check for valid cookie on all pages!
+    Return True if there is valid cookie.
 
-''' cookie_check function will check for valid cookie on all pages!
-Return True if there is cookie with requested name.
-'''
-def cookie_check(request):
+    """
     cookie = request.cookies.get(COOKIE_NAME, None)
-    if cookie:
-        if COOKIE_NAME == COOKIE_NAME:
-            return True
-        # Normal test for checking the COOKIE_NAME
-        else:
-            print("COOKIE_NAME has been modified!.")
-    # Return False when there is no cookie at all.
-    else:
+    if not cookie:
+        return False
+    if COOKIE_VALUE == 0:
+        return True
```

Comment:
Here the function can check for cookie with name and value! but somehow, in future, the name and value can be changed.